### ISSUE TITLE
Code that makes the engine panic

### ISSUE THREAD

**Issue by chaosprint** [chaosprint/glicol#38]
```
lead: saw ~several

~several: squ 0.2 >> mul 200 >> add 800 >> mul ~more

~more: choose 60 50 40
```

**Comment by chaosprint** [comment]
The problem is caused by the output range of `choose`
The chain `~several` may output a very large number.
But the `saw` node should be able to handle this large number, isn't it?

**Comment by chaosprint** [comment]
fixed with #49
